pages/common/views/components/form.jsx

```diff
@@ -1,34 +1,57 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import { map, get } from 'lodash';
 import { TextArea, Input, RadioGroup, Select } from 'govuk-react-components';
 import Snippet from '../containers/snippet';
+import ConditionalReveal from './conditional-reveal';
 
 const fields = {
   inputText: ({ ...props }) => <Input { ...props } />,
-  textarea: ({ value, ...props }) => <TextArea { ...props }>{ value }</TextArea>,
+  textarea: ({ value, ...props }) => <TextArea value={value} { ...props } />,
   radioGroup: ({ ...props }) => <RadioGroup { ...props } />,
   checkboxGroup: ({ ...props }) => <RadioGroup type="checkbox" { ...props } />,
-  select: ({ ...props }) => <Select { ...props } />
+  select: ({ ...props }) => <Select { ...props } />,
+  text: props => props.value &&
+    <Fragment>
+      <h3>{ props.label }</h3>
+      <p>{ props.value }</p>
+    </Fragment>
```

Comment:
We might want to consider an "arbitrary content" element, that just takes another component as a prop.

Reply:
like a `Fragment` :)